main.qml

```diff
@@ -1413,6 +1414,12 @@ ApplicationWindow {
     StandardDialog {
         z: parent.z + 1
         id: transactionConfirmationPopup
+        dialogHeight: 380
+        icon: StandardIcon.Question
+        cancelText: qsTr("Back") + translationManager.emptyString
+        cancelButton.width: okButton.width
+        okButton.rightIcon: "qrc:///images/rightArrow.png"
```

Comment:
Use FontAwesome icon. For example, solid `arrowCircleRight` or `checkCircle`.

Reply:
FontAwesome doesn't work with `.rightIcon`

Also, if we change button text to `qsTr("Confirm") + FontAwesome.checkCircle`, the FontAwesome isn't displayed on Windows (see #2544).

If we try the solution of #2556 (change button fontFamily to FontAwesome.fontFamilySolid), the button text has a font that is bold and not so nice, see below:
![image](https://user-images.githubusercontent.com/45968869/73503890-cc1c8800-43cd-11ea-8578-e2b8678da6ad.png)

I used the same png icon as the send button, but maybe we could create a png icon with a checkCircle image. Also, if it is possible, we could make the button use two font families, but I don't know if it is possible. Any other idea?

Reply:
Gotcha, let's use `rightArrow.png` for now and update it to `FontAwesome.checkCircle` later.

Reply:
I decided to add a checkCircle.png as icon